src/components/dashboardPanel/dashboardPanel.scss

```diff
@@ -3,8 +3,6 @@
 $padding: 15px;
 
 .dashboard-panel {
-  display: flex;
```

Comment:
So I looked into this. Don't remove these values, instead add `width: 100%;` to the `> .panel-contents`. That will fix the issue in Edge.